rust/numaflow-core/src/message.rs

```diff
@@ -7,9 +7,9 @@ use chrono::{DateTime, Utc};
 use crate::error::Error;
 use crate::monovertex::sink_pb::sink_request::Request;
 use crate::monovertex::sink_pb::SinkRequest;
-use crate::monovertex::{source_pb, sourcetransform_pb};
 use crate::monovertex::source_pb::{read_response, AckRequest};
 use crate::monovertex::sourcetransform_pb::SourceTransformRequest;
+use crate::monovertex::{source_pb, sourcetransform_pb};
```

Comment:
Just curious, I saw a lot of formatting changes in the PR, how did you generate those? asking because I ran `cargo fmt` in my last PR and it didn't generate your changes. is it because we are using different formatting scheme?

Reply:
i do "cargo fmt" right before committing.